Decred Wallet/Features/More/More Features/Debug/StatisticsViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  StatisticsViewController.swift
+//  Decred Wallet
+//
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+//
+
+import Foundation
+import UIKit
+
+class StatisticsViewController: UITableViewController  {
+    @IBOutlet weak var buildDetailLabel: UILabel!
+    
+    @IBOutlet weak var peerConnectedDetailLabel: UILabel!
+    @IBOutlet weak var uptimeDetailLabel: UILabel!
+    @IBOutlet weak var networkDetailLabel: UILabel!
+    @IBOutlet weak var bestBlockDetailLabel: UILabel!
+    @IBOutlet weak var bestBlockTimestampDetailLabel: UILabel!
+    
+    @IBOutlet weak var bestBlockAgeDetailLabel: UILabel!
+    @IBOutlet weak var walletFileDetailLabel: UILabel!
+    @IBOutlet weak var chainDataDetailLabel: UILabel!
+    
+    @IBOutlet weak var transactionDetailLabel: UILabel!
+    @IBOutlet weak var accountDetailLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.loadStatistcs()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+       self.navigationController?.navigationBar.isHidden = false
+        self.navigationController?.navigationBar.tintColor = UIColor.appColors.darkBlue
+        self.navigationController?.navigationBar.barTintColor = UIColor.appColors.offWhite
+        //Remove shadow from navigation bar
+        self.navigationController?.navigationBar.setBackgroundImage(UIImage(), for: UIBarMetrics.default)
+        self.navigationController?.navigationBar.shadowImage = UIImage()
+        
+        self.addNavigationBackButton()
+            
+        let barButtonTitle = UIBarButtonItem(title: LocalizedStrings.statistics, style: .plain, target: self, action: nil)
+        barButtonTitle.tintColor = UIColor.appColors.darkBlue
+            
+        self.navigationItem.leftBarButtonItems =  [(self.navigationItem.leftBarButtonItem)!, barButtonTitle]
+    }
+    
+    func loadStatistcs() {
+        let dateformater = DateFormatter()
+        dateformater.dateFormat = "yyyy-MM-dd"
```

Comment:
Can we have an extension on the dateFormatter that will return a string given a date please!